pkgs/development/python-modules/solo-python/default.nix

```diff
@@ -11,7 +12,17 @@
 , pyusb
 , requests
 }:
-
+let
+  # https://github.com/solokeys/solo-python/issues/110#issuecomment-802945287
+  fido2_0_8_1 = fido2.overridePythonAttrs(oldAttrs: rec {
```

Comment:
This is a red flag to me. Policy is to not override any python packages in the default package set, because that can cause package conflicts if e.g. someone tries to have a python environment with e.g. ``solo-python`` and ``fido2`` in the same environment.

If you're just overriding this for purposes of ``yubikey-manager``, you could try something along the lines of ``conan``: https://github.com/NixOS/nixpkgs/blob/17c66529f686d2138f7a5fdcc66f8cf6c7e052a5/pkgs/development/tools/build-managers/conan/default.nix.

Otherwise, maybe try unpinning the ``fido2`` dependency in ``solo-python``? Since the comment that you linked says that ``solo-python`` isn't being updated soon, maybe we just have to take the hit and mark ``solo-python`` as broken. @jonringer, thoughts?

Reply:
Thanks for your comment. In fact, I was expecting some hesitation regarding this change but I couldn't think of a proper solution on how to deal with this other than marking `solo-python` as broken. If this is acceptable, I'm happy to do that.

Reply:
+1 for making `solo-python` broken for now with nice comment explaining where it should be marked as unbroken again

Reply:
Done! Thanks for reviewing this so quickly, @drewrisinger and @prusnak!